manifests/cf-manifest/operations.d/201-cf-upgrade-cflinuxfs2.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- type: replace
+  path: /releases/name=cflinuxfs2
+  value:
+    name: cflinuxfs2
+    url: https://bosh.io/d/github.com/cloudfoundry/cflinuxfs2-release?v=1.218.0
+    version: 1.218.0
+    sha1: "4a59435d047dd7e9683974a602f5ee7271ef3d70"
```

Comment:
We're already using version 1.219.0 as specified in https://github.com/alphagov/paas-cf/blob/master/manifests/cf-manifest/operations.d/300-upgrade-cflinuxfs2-for-cve-mitigation.yml (from #1398)